webroot/src/components/CollapseCaretButton/CollapseCaretButton.less

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
_UI testing find - just commenting here for a resolvable comment_

On the purchase screen, I noticed that any server errors are displayed at the top of the page. It took me a sec to realize that because that area of the page was often scrolled out of my viewport by the time I clicked the submit button. Up to now, we've been putting errors like that in the built-in message UI of the submit button - using the `this.setError(msg)` of components that extend the `MixingForm`. You can navigate `form.mixin.ts` to familiarize. You don't have to use the message area of the submit if it doesn't fit here, but maybe put the message just above the row with the submit buttons.

<img width="932" alt="Screenshot 2024-11-11 at 10 32 59 AM" src="https://github.com/user-attachments/assets/43d44af9-2b8d-41a0-bb88-b16ba92c6148">